content/docs/user-guide/packaging/project-export/project-export-ios.md

```diff
@@ -0,0 +1,101 @@
+---
+linkTitle: Project Export iOS
+title: Project Export for iOS
+description: (Experimental) Learn how to use the Project Export CLI to create an Xcode project to build and deploy for iOS.
+toc: true
+weight: 450
+---
+This guide covers how to use project export tooling to build and deploy games on iOS. First, we will use the iOS export script to generate an Xcode project and prepare assets. Afterwards we will walk through how to ensure Xcode is properly setup to build an iOS app.
+
+For this demonstration, we will use the O3DE Atom Sample Viewer project as our reference point for ensuring proper behavior. Results may vary for other projects.
+
+{{< note >}}
+To learn more about the project export tooling, please consult the page [Project Export CLI Tool.](project-export-cli)
```

Comment:
Broken link. Relative paths work differently inside `note` shortcodes -- as if the note were one directory level deeper. You will need to either add a `../` in front of this path or use an absolute path.
```suggestion
To learn more about the project export tooling, please consult the page [Project Export CLI Tool.](/docs/user-guide/packaging/project-export/project-export-cli)
```